x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/AsyncOperator.java

```diff
@@ -33,6 +35,7 @@ public abstract class AsyncOperator implements Operator {
     private final DriverContext driverContext;
 
     private final int maxOutstandingRequests;
+    private final LongAdder totalTimeInNanos = new LongAdder();
```

Comment:
Oh fun, a place where I think we actually want a multithreaded timer!

Reply:
Yeah, I've considered replacing these async operators with an exchange instead. I'm not certain if it would improve performance, but I plan to give it a try sometime.